### PR TITLE
Progress on private method Javadocs

### PR DESCRIPTION
**Pull request by melissalinkert**
See https://trello.com/c/U8HDtN3O/159-review-private-method-javadocs.  This only covers the ```bio-formats-tools```, ```metakit```, and ```formats-api``` components; I likely won't have time to review others before 5.1.2.

Please focus on reviewing the readability and correctness of what's here; if a method is undocumented, it can be listed on the card, but the focus at this point is not on an exhaustive review of all private methods.

### CODE REVIEW

File: components/bio-formats-tools/src/loci/formats/tools/ImageConverter.java [hunk 1]
```diff
@@ -582,6 +590,16 @@ else if (writer instanceof ImageWriter) {
 
   // -- Helper methods --
 
+  /**
+   * Convert the specified plane using the given writer.
+   * @param writer the loci.formats.IFormatWriter to use for writing the plane
```
Comment (mtbc) [inline components/bio-formats-tools/src/loci/formats/tools/ImageConverter.java]: maybe `@link` for this kind of thing?

File: components/formats-api/src/loci/formats/FormatReader.java [hunk 2]
```diff
@@ -1897,6 +1927,11 @@ protected Pulse getPulse(String value) throws FormatException {
     }
   }
 
+  /**
+   * Construct an {@link ome.xml.model.AffineTransform} corresponding to
+   * the given angle.
+   * @param theta the angle of rotation
```
Comment (mtbc) [inline components/formats-api/src/loci/formats/FormatReader.java]: mention units?

**Comment by mtbc** [comment]
:+1: Already good to merge as a clear improvement.

(Might consider some use of `@link` where literal class names are used, depends on context I guess.)

**Comment by melissalinkert** [comment]
Not quite good to merge given that this introduces warnings per https://ci.openmicroscopy.org/view/Bio-Formats/job/BIOFORMATS-5.1-merge-build/867.  Will push a fix for that and above comments shortly.

**Comment by melissalinkert** [comment]
Fixes pushed - this should hopefully fix the build this evening.

**Comment by sbesson** [comment]
https://ci.openmicroscopy.org/view/Failing/job/BIOFORMATS-5.1-merge-build/868/ is green.

**Pull request merged**